### ISSUE TITLE
non-ascii characters 

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#3334]
Check for non-ascii characters in feature names - CHECK FAILURE:  expected 0 but got 3
uniquename	name	session
  SPAC20G4.02c:allele-16	pfus1:fus1N-cdc12Δolig	345d6b0c8bf7e493
  SPCC895.07:allele-12	alp14Δmid	f134593392c70d97
  SPCC895.07:allele-13	alp14ΔtubBD	f134593392c70d97

**Comment by manulera** [comment]
Why do you think these are flagged @kimrutherford ? Related issue where same thing was happenning for the same alleles:https://github.com/pombase/curation/issues/3324

When they first came up, I replaced the delta in their name, by copy-pasting the delta from other alleles in the same session that were not being flagged, and the problem persisted.

Then I think the problem went away, but now it came back for the same alleles. Perhaps it's not even the delta character and there is some weird space or something in them?

**Comment by ValWood** [comment]
For Δ we always write "delta" and  it is interpreted on the web pages as Δ

**Comment by manulera** [comment]
But it cannot be only that, because in the same session, there is `fus1ΔCter` spelled with the delta, and it does not raise this error:

https://curation.pombase.org/pombe/curs/345d6b0c8bf7e493#

**Comment by ValWood** [comment]
weird. Id that allele in any other session?

**Comment by manulera** [comment]
No, neither the fus1deltaCter. Also, now that I look at the publication entry in PomBase:
https://www.pombase.org/reference/PMID:35673994

This `fus1ΔCter` is not even there, and the session was approved a while ago (you can see that it is not in the last pombase helpdesk updates).

**Comment by ValWood** [comment]
OK @kimrutherford  will need to answer..

**Comment by kimrutherford** [comment]
> OK @kimrutherford will need to answer..

The main cause of this weirdness is my bad code.

The code that loads the Canto JSON into Chado automatically converts "Δ" to "delta" in allele names.  But when I implemented that conversion I was too conservative and decided only to convert the deltas at the ends of names.  Then I got the regular expression wrong so it was converting when the "Δ" was followed by an uppercase letter but not when followed by a lowercase letter.

Anyway, I shouldn't have been that conservative so I've fixed the nonsense code to always replace the "Δ".  I didn't get the fix done in time for the current load.  It should be fixed on Tuesday morning.

(We store "Δ" as "delta" in Chado but the website code displays "Δ" instead of "delta" so it shouldn't change how things are displayed)
